2-ui/4-forms-controls/4-forms-submit/article.md

```diff
@@ -10,7 +10,7 @@ Let's see more details of them.
 
 There are two main ways to submit a form:
 
-1. The first -- to click `<input type="submit">` or `<input type="image">`.
+1. The first -- to click `<input type="submit">` or `<button>` or `<input type="image">`.
```

Comment:
```suggestion
1. The first -- to click `<input type="submit">`, `<input type="image">` or `<button type="submit">`.
```

Reply:
@bogdanbacosca the submit event seems to be triggered even when clicking on a <button> without type="submit", as long as it is inside the form. So should I use this instead?
```suggestion
1. The first -- to click `<input type="submit">`, `<input type="image">` or `<button>`.
```

Reply:
that should be fine also